ocaml/typing/jkind.mli

```diff
@@ -177,12 +177,14 @@ end
 type sort = Sort.t
 
 (* The layout of a type describes its memory layout. A layout is either the
-   indeterminate [Any] or a sort, which is a concrete memory layout. *)
+   indeterminate [Any], a sort, which is a concrete memory layout, or
+   [Non_null_value], which is the same as [Value] but also can't be null. *)
```

Comment:
I think I would say "...which is a sublayout of the sort [Value] describing types that do not allow the concrete value null".  And here or somewhere else it should be noted that this is the layout of classic ocaml values.

Reply:
Done.